src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -0,0 +1,216 @@
+import _ from 'underscore';
+import hbs from 'handlebars-inline-precompile';
+import Radio from 'backbone.radio';
+import { View } from 'marionette';
+
+import InputWatcherBehavior from 'js/behaviors/input-watcher';
+import Droplist from 'js/components/droplist';
+
+import intl from 'js/i18n';
+
+import 'sass/modules/buttons.scss';
+import 'sass/modules/forms.scss';
+import 'sass/modules/sidebar.scss';
+
+import ClinicianSidebarTemplate from './clinician-sidebar.hbs';
+
+const RoleItemTemplate = hbs`<a{{#if isSelected}} class="is-selected"{{/if}}>{{matchText name query}} <span class="clinician__role">{{matchText short query}}</span></a>`;
+
+const RoleComponent = Droplist.extend({
+  isCompact: false,
+  getTemplate() {
+    if (this.getOption('isCompact')) {
+      return hbs`{{far "user-circle"}}{{ short }}`;
+    }
+
+    return hbs`{{far "user-circle"}}{{ name }}{{#unless name}}{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.roleComponent.roleDefaultText }}{{/unless}}`;
+  },
+  popWidth() {
+    const isCompact = this.getOption('isCompact');
+
+    return isCompact ? null : this.getView().$el.outerWidth();
+  },
+  picklistOptions() {
+    return {
+      isSelectlist: true,
+    };
+  },
+  viewOptions() {
+    const isCompact = this.getOption('isCompact');
+    const selected = this.getState('selected');
+    return {
+      modelEvents: {
+        'change:_role': 'render',
+      },
+      className() {
+        if (!selected && isCompact) {
+          return 'button-secondary--compact is-icon-only';
+        }
+
+        if (isCompact) {
+          return 'button-secondary--compact';
+        }
+
+        return 'button-secondary w-100';
+      },
+      template: this.getTemplate(),
+    };
+  },
+  initialize({ model }) {
+    const currentOrg = Radio.request('bootstrap', 'currentOrg');
+    const roles = currentOrg.getActiveRoles();
+
+    this.lists = [{
+      collection: roles,
+      itemTemplate: RoleItemTemplate,
+      headingText: intl.admin.sidebar.clinician.clinicianSidebarViews.roleComponent.rolesHeadingText,
+      attr: 'name',
+    }];
+
+    const role = Radio.request('entities', 'roles:model', model.get('_role'));
+    this.setState({ selected: role });
+  },
+  onChangeSelected(selected) {
+    this.triggerMethod('change:role', selected);
+  },
+});
+
+const NameView = View.extend({
+  className: 'pos--relative',
+  template: hbs`<input class="input-primary w-100 js-input"{{#if isNew}} placeholder="{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.nameView.placeholder }}"{{/if}} value="{{ name }}" />`,
+  behaviors: [InputWatcherBehavior],
```

Comment:
No need to key watch.  Can just do `'input .js-input` in events

Reply:
Don't we need the watcher to determine when to enable/disable the save view?

Reply:
`onWatchKeydown` I don't think should be needed?

So the behavior then I believe is equivalent to:
```js
events: {
  'input @ui.input`: 'onInput',
},
onInput() {
  this.model.set('name', this.ui.input.val());
},
```
I suppose we could keep it for consistency?  But I don't think the keydown watch is necessary for an input type=text?